pkg/apis/cluster/v1alpha1/conversions/testdata/machinesv1alpha1machine/linode.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: "machine.k8s.io/v1alpha1"
+kind: Machine
+metadata:
+  name: linode
+spec:
+  metadata:
+    name: node1
+  providerConfig:
+    sshPublicKeys:
+      - "<< YOUR_PUBLIC_KEY >>"
+    cloudProvider: "linode"
+    cloudProviderSpec:
+      token: "token"
+      region: eu-west
+      size: g6-standard-2
```

Comment:
These fixtures are not really accurate, the `Linode` ProviderSpec doesn't have a `size` property. Maybe just remove them?

Reply:
The Linode value that makes sense here is `type` (instance type), which I've paralleled to size in other projects.
g6-nanode-1 (1gb ram, 1vcpu, 25gb SSD)
g6-standard-2 (..)
...
g6-highmem-16 (...)
...
g6-dedicated-1 (...)
...